packages/jest-cli/src/SearchSource.js

```diff
@@ -142,8 +143,9 @@ class SearchSource {
   _getAllTestPaths(
     testPathPattern: StrOrRegExpPattern,
   ): SearchResult {
+    const allFiles = this._hasteContext.hasteFS.getAllFiles();
     return this._filterTestPathsWithStats(
-      this._hasteContext.hasteFS.getAllFiles(),
+      allFiles,
```

Comment:
can you revert the changes to this method? They are now unrelated to the change you are making and we can simply inline the function call again :)